src/constants.js

```diff
@@ -69,7 +69,7 @@ module.exports = {
     /**
      * The telemetry endpoint to send data to.
      */
-    TELEMETRY_ENDPOINT: "https://cors-anywhere.herokuapp.com/https://incoming.telemetry.mozilla.org/",
```

Comment:
Changing this all the time is also not beautiful, when this project is not a POC we can surely base this in a .env file instead that is ignored by git or something of the sort.